### DISCUSSION TITLE
MPAS-A version 8.0.1

### DISCUSSION THREAD

**Discussion by daniloceano** [pedrospeixoto/MPAS-PXT#5] [Ideas]
Olá pessoal,

Vocês devem ter recebido o anúncio do Michel Duda que a versão 8.0.1 ofi lançada. Creio que atualmente o repositório está utilizando versão 7.2 (?). Não seria interessante tentarmos atualizar? Não sei como o pontapé inicial seria dado, mas uma vez que isso fosse feito, eu posso auxiliar no processo

**Comment by pedrospeixoto** [comment]
Olá,

Seria importante atualizarmos a versão. Felipe, você quer tentar fazer a
atualização?

Abrs,

Prof. Dr. Pedro da Silva Peixoto
Associate Professor
Applied Mathematics
University of São Paulo (USP)
www.ime.usp.br/~pedrosp



Em sáb., 8 de jul. de 2023 01:25, Danilo Couto de Souza <
***@***.***> escreveu:

**Comment by favba** [comment]
Olá pessoal,

Criei um novo branch chamado `develop` onde fiz um git merge do branch de mesmo nome do MPAS-Dev com nosso master.
Então este branch tem todas nossas modificações com a versão develop do MPAS, que é a v8.0.1 adicionado dos últimos desenvolvimentos feito no MPAS-Dev que não foram lançados numa nova versão ainda mas já estão publicados.

@daniloceano , antes de fazer o merge eu fiz uma limpeza do master, onde deletei todos os arquivos `INMET_*.CSV` pois achei que seria melhor não misturar dados com o código fonte no repositório. Acho melhor criarmos um script que faz o download dos dados (como fazemos com a malha, WRF physics, ERA5...). Tudo bem por você? Se sim, você poderia fazer este script? Ou então me apontar onde você fez o download para eu fazer o script.

Eu testei só a compilação do código neste novo branch por enquanto e funcionou bem. @daniloceano, talvez você queira testar seus benchmarks neste branch e ver se está tudo funcionando ainda

**Comment by daniloceano** [comment]
@favba obrigado!

Tudo bem remover os arquivos. Porém, infelizmente, não é possível automatizar o download dos mesmos, pois o sistema do INMET ainda está na idade da pedra.

Estamos reestruturando o sistema (e, consequentemente as máquinas) aqui do MASTER-IAG, mas assim que tudo for estabilizado eu testo os benchmakrs.

**Reply by favba** [reply]
Os dados do INMET que estão no master branch são os mesmo que estes [aqui](https://portal.inmet.gov.br/dadoshistoricos)?
Se for posso criar o script.

**Reply by daniloceano** [reply]
Eles mesmos!

**Comment by favba** [comment]
O master agora está na versão 8.0.1!

@daniloceano, também criei um branch chamado `master_before_merge` que tem o código como estava antes de atualizar para versão 8. Então se você tiver algum problema com o master agora você pode `git checkout master_before_merge` que o código estará do mesmo jeito de quando você estava rodando.

**Reply by daniloceano** [reply]
Obrigado @favba !
